pkg/kapp/clusterapply/exists_change.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package clusterapply
+
+import (
+	"errors"
+
+	ctldiff "github.com/k14s/kapp/pkg/kapp/diff"
+	ctlres "github.com/k14s/kapp/pkg/kapp/resources"
+)
+
+type ExistsChange struct {
+	change              ctldiff.Change
+	identifiedResources ctlres.IdentifiedResources
+}
+
+func (c ExistsChange) ApplyStrategy() (ApplyStrategy, error) {
+	res := c.change.NewResource()
+	return WaitStrategy{res, c}, nil
+}
+
+type WaitStrategy struct {
+	res ctlres.Resource
+	d   ExistsChange
```

Comment:
Why not name it `change` ?

```suggestion
	change   ExistsChange
```

Reply:
We already have a `change` variable in `ExistsChange`. But it makes moree sense to use `e` than using `d`. I will update it.